src/services/about.rs

```diff
@@ -1,17 +1,30 @@
 extern crate reqwest;
 
 use crate::config::{base_transaction_service_url};
-use crate::models::backend::about::About;
+use crate::models::backend::about::About as AboutDto;
 use anyhow::Result;
 use reqwest::Url;
+use crate::models::service::about::About;
+use std::process::Command;
 
-pub fn get_about() -> Result<String> {
+pub fn get_about() -> Result<About> {
     let url_string = format!("{}{}", base_transaction_service_url(), "/about");
     let url = Url::parse(&url_string)?;
     let body = reqwest::blocking::get(url)?.text()?;
-    let about: About = serde_json::from_str(&body)?;
-    Ok(format!(
-        "This is an API wrapper for {}, version {}\nNo guarantees in terms of availability.",
-        about.name, about.api_version
-    ))
+    let about_dto: AboutDto = serde_json::from_str(&body)?;
+
+    let command = Command::new("git")
```

Comment:
This should probably be a value that is set on compile time by the ci (as we cannot assume that the web worker has git or the git history). Maybe we should leave this out for now